src/SablierV2LockupDynamic.sol

```diff
@@ -240,13 +240,13 @@ contract SablierV2LockupDynamic is
             currentSegmentTimestamp = segments[index].timestamp;
 
             uint40 previousTimestamp;
-            if (index > 0) {
+            if (index == 0) {
+                // Otherwise, the current segment is the first, so use the start time as the previous timestamp.
```

Comment:
Wording needs to be changed here - "Otherwise" should be moved under the `else`.